common/utils/src/main/resources/error/error-classes.json

```
@@ -921,6 +921,11 @@
     ],
     "sqlState" : "42809"
   },
+  "FUNCTION_AND_ORDER_EXPRESSION_MISMATCH_ERROR" : {
```

All error classes are errors, the suffix `_ERROR` doesn't give any additional info. Could you remove it, please:
```suggestion
  "FUNCTION_AND_ORDER_EXPRESSION_MISMATCH" : {
```